src/ClientCommand.cpp

```diff
@@ -438,6 +438,50 @@ void CClient::UserCommand(CString& sLine) {
                       "] channels matching [" + sPatterns + "]");
             PutStatus("Disabled [" + CString(uDisabled) + "] channels");
         }
+    } else if (sCommand.Equals("MOVECHAN")) {
+        if (!m_pNetwork) {
+            PutStatus("You must be connected with a network to use this command");
+            return;
+        }
+
+        const auto sChan = sLine.Token(1);
+        const auto sTarget = sLine.Token(2);
+        if (sChan.empty() || sTarget.empty()) {
+            PutStatus("Usage: MoveChan <#chan> <index>");
+            return;
+        }
+
+        // use one-based indexing
+        unsigned int uIndex = sTarget.ToUInt();
+        if (uIndex == 0 || uIndex > m_pNetwork->GetChans().size()) {
+            PutStatus(t_f("Invalid index {1}")(uIndex));
```

Comment:
It's not obvious what number user should pass to the command. Probably ListChans should show the current index?